workflows/tests/test_ena_api_requests.py

```diff
@@ -122,6 +122,39 @@ def test_get_study_from_ena_use_secondary_as_primary(httpx_mock, prefect_harness
     assert len(created_study.additional_accessions) == 1
 
 
+@pytest.mark.httpx_mock(should_mock=should_not_mock_httpx_requests_to_prefect_server)
```

Comment:
I *think* in this case you won't need the `should_mock` filter, as no prefect server test harness fixture is running here. But it is also fine to leave it as it does no harm :) 

```suggestion
@pytest.mark.httpx_mock
```

Reply:
Yes, you're right. I just copied it without thinking...